### PR TITLE
WT-12597 Track the number of pages read into cache by checkpoint

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

**Review by korteland — APPROVED** [review]
LGTM - I assume we're backporting this?

File: dist/stat_data.py [hunk 1]
```diff
@@ -1015,6 +1015,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_prefetch', 'pages requested from the cache due to pre-fetch'),
     CacheStat('cache_pages_requested', 'pages requested from the cache'),
     CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_checkpoint', 'pages read into cache by the checkpoint'),
```
Comment (korteland) [inline dist/stat_data.py:1018]: Nitpick: `by the checkpoint` -> `by checkpoint`

**Review by etienneptl — APPROVED** [review]

File: dist/stat_data.py (hunk 1 above)
Comment (kommiharibabu) [reply dist/stat_data.py:1018]: Fixed.

**Comment by kommiharibabu** [comment]
@korteland Yes, we want to backport it to understand the cause of the long running checkpoint.

**Pull request merged**